packages/react-dev-overlay/package.json

```diff
@@ -30,7 +30,6 @@
   },
   "peerDependencies": {
     "react": "^16.9.0 || ^17",
-    "react-dom": "^16.9.0 || ^17",
-    "webpack": "^4 || ^5"
+    "react-dom": "^16.9.0 || ^17"
```

Comment:
Looks like we need to eslint-ignore the webpack require to make eslint ok with this change: 
```sh
/home/runner/work/next.js/next.js/packages/react-dev-overlay/src/middleware.ts
  14:1  error  'webpack' should be listed in the project's dependencies. Run 'npm i -S webpack' to add it  import/no-extraneous-dependencies
```